modules/power-monitoring-fips-support.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * power_monitoring/power-monitoring-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="power-monitoring-fips-support_{context}"]
+= About FIPS compliance for {PM-operator}
+
+Starting with version 0.4, {PM-operator} for Red Hat OpenShift is FIPS compliant. When deployed on an {product-title} cluster in FIPS mode, it uses {op-system-base-full} cryptographic libraries validated by National Institute of Standards and Technology (NIST).
+
+For details on the NIST validation program, see link:https://csrc.nist.gov/Projects/cryptographic-module-validation-program/validated-modules[Cryptographic module validation program]. For the latest NIST status of {op-system-base} cryptographic libraries, see link:https://access.redhat.com/en/compliance[Compliance activities and government standards].
+
+To enable FIPS mode, you must install {PM-operator} for Red{nbsp}Hat OpenShift on an {product-title} cluster. For more information, see "Do you need extra security for your cluster?"
```

Comment:
```suggestion
To enable FIPS mode, you must install {PM-operator} for Red{nbsp}Hat OpenShift on an {product-title} cluster. For more information, see "Do you need extra security for your cluster?".
```

Reply:
Addressing the changes in: https://github.com/openshift/openshift-docs/pull/89583